uploader/download.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const downloader = require("./downloader");
+
+module.exports = function(version) {
```

Comment:
I'd be inclined to call this module something like `downloadAll` not just `download`. It's a bit weird when it's consumed and you write `downloads = download(version)`.

Reply:
Should this be reflected in the filename as well as documentation?